docs/get-started.md

```diff
@@ -1,42 +1,120 @@
-## Get Started
+# Get Started with Open Service Mesh Arc extension
 Use the following scripts to publish chart, enable connected cluster, and add OSM Arc extension.
 
 ## Prerequisites
 - Running Kubernetes cluster
-- Azure CLI
+- [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
-### Set env vars
-The OSM Azure repo relies on environment variables to make it usable on your localhost. The root of the repository contains a file named `.env.example`. Copy the contents of this file into `.env`
+## Add Azure CLI extensions
+
+> Note: Get latest wheels: https://github.com/Azure/azure-arc-kubernetes-preview/tree/master/extensions
+
+1. Install the latest [connectedk8s](https://docs.microsoft.com/en-us/cli/azure/connectedk8s?view=azure-cli-latest).
 ```bash
-cat .env.example > .env
+WHEEL="https://arcextensions.blob.core.windows.net/extensions/connectedk8s-0.3.8-py2.py3-none-any.whl"
```

Comment:
I think this should not be exposed. let's discuss this offline

Reply:
Removed